samples/net/run-sample.sh

```diff
@@ -0,0 +1,320 @@
+#!/bin/sh
```

Comment:
Please put this script under scripts/, this is no place for scripts, for example scripts/net/...

Reply:
Yes, done.